### ISSUE TITLE
Investigation: Create Flappy Bird

### ISSUE THREAD

**Issue by jcollard** [GameEngineDesign-S24/AstraEngine#18]
Is it possible to create a Flappy Bird clone?

What features would make it possible / easier to implement?

**Comment by maxwaldman8** [comment]
No, it is not fully possible yet, as you would need a sprite renderer to render the bird. Also, it would be useful to have a method to check whether a key is being pressed for the first time (something like IsKeyPressed()) for the jumping. There are also probably other things that are needed that I haven't ran into yet (collision detection for example).
